scripts/publish.js

```diff
@@ -65,6 +65,15 @@ async function run() {
     // If package is based on a previous published version
     if (argv.fromTag) {
 
+
+        // Clean up .git folders. NPM will fail
+        // npm ERR! code EISGIT
+        // npm ERR! git /home/travis/build/centrifuge/centrifuge-ethereum-contracts/node_modules/web3-providers-ws/node_modules/websocket: Appears to be a git repo or submodule.
+        await execute(`rm -rf node_modules/**/.git`);
```

Comment:
`find . \( -name ".git" \) -exec rm -rf -- {} +`
to avoid having the two rm statements